lib/control-app.js

```diff
@@ -25,10 +41,15 @@ module.exports = function(config) {
 
     var build;
     // default builder is browserify which we provide
-    config.builder = config.builder || '../lib/builder-browserify';
+    config.builder = config.builder || defaultBuilder;
```

Comment:
Allright, I [updated the PR](https://github.com/vvo/zuul/commit/a64da244372d3364d8243410242a5c001ecd908e#diff-20cb57a5653789fb88cffd4a02f0b95fR46), no prob.